agent/__init__.py

```
@@ -228,16 +206,23 @@ def get_ca_cert(debug=False):
     return ca.json()['ca_bundle']
 
 
-def get_claim_token():
-    config = configparser.ConfigParser()
-    config.read(INI_PATH)
-    return config['DEFAULT'].get('claim_token', None)
+def get_claim_info(debug=False, dev=False):
+    setup_endpoints(dev, debug)
```

Need to call `can_read_cert()` before using MTLS to fail fast.